From 1c8b76caab7d5164ac2d0d09aff9e4ffecdf205e Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 9 Mar 2015 18:01:47 +0100
Subject: [PATCH] man: document that ExecStartPre= is not the place to start
 long-running processes

(cherry picked from commit b481de3b22fcd838a8f059aed8745375afdb9eca)
---
 man/systemd.service.xml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/man/systemd.service.xml b/man/systemd.service.xml
index c03b4e8a5..f59870563 100644
--- a/man/systemd.service.xml
+++ b/man/systemd.service.xml
@@ -334,6 +334,11 @@
         <para>If any of those commands (not prefixed with
         <literal>-</literal>) fail, the rest are not executed and the
         unit is considered failed.</para>
+
+        <para>Note that <varname>ExecStartPre=</varname> may not be
+        used to start long-running processes. All processes forked
+        off by processes invoked via <varname>ExecStartPre=</varname> will
+        be killed before the next service process is run.</para>
         </listitem>
       </varlistentry>
 
